server/src/main/kotlin/ir/armor/tachidesk/server/util/SystemTray.kt

```diff
@@ -10,16 +10,17 @@ package ir.armor.tachidesk.server.util
 import dorkbox.systemTray.MenuItem
 import dorkbox.systemTray.SystemTray
 import dorkbox.systemTray.SystemTray.TrayType
-import dorkbox.util.CacheUtil
-import dorkbox.util.Desktop
```

Comment:
the library still has this package in it's tests, are you sure it's removed?
https://github.com/dorkbox/SystemTray/blob/6b4893b223312cbecc60d1544a97c48b85ee2a22/test/dorkbox/TestTray.java#L30

Reply:
Ya I checked, there is no alternatives in the library. He probably hasnt updated the tests. If you look at the actual code its not there.

Reply:
It has never been there(I checked commits from 2017-18) it seems to be added on build time.

Reply:
I think I found it, its a implementation thats not visible to projects that just implement the system tray library now
implementation("com.dorkbox:Utilities:1.9")